rpcs3/Emu/Io/usb_device.cpp

```diff
@@ -34,18 +36,47 @@ u64 usb_device::get_timestamp()
 	return (get_system_time() - Emu.GetPauseTime());
 }
 
+bool usb_device::get_device_location(u8* location)
+{
+	if (!location)
+		return false;
+
+	// for the dummy device we just give it a location of '0'
+	location[0] = 0;
+	location[1] = 0;
+	location[2] = 0;
+	location[3] = 0;
+	location[4] = 0;
+	location[5] = 0;
+	location[6] = 0;
```

Comment:
How do we know it's seven long?

Reply:
That's the USB 2.0 bus topology maximum depth, it also happens to be the USB 3.0 maximum depth as well (so it aligns with libusb).  I've tidied this up a bit by using memset so that the depth is tied to the enum value of MAX_USB_PATH_LEN = 6